src/dto/test_results/TestResultGherkinTestRunElement.ts

```diff
@@ -0,0 +1,12 @@
+import {DtoObject} from "../DtoObject";
+import {TestResultGherkinTestRunAttributes} from "./TestResultGherkinTestRunAttributes";
+import {TestResultTestRunElement} from "./TestResultTestRunElement";
+
+export class TestResultGherkinTestRunElement extends TestResultTestRunElement {
+    gherkin_test_run: any;
```

Comment:
no access modifier? And where did you get_this_pattern_from? We should use the proper naming, regardless of the current state of the code: https://makecode.com/extensions/naming-conventions